### DISCUSSION TITLE
Question about the design document (设计文档)

### DISCUSSION THREAD

**Discussion by hoho1221** [apache/iotdb#5742] [General]
设计文档的「分布式」章节中，部分文档内容较之前有删减，且当前高频开发的查询框架似乎也没有完善文档，请考虑完善下，thx

**Comment by jixuan1989** [comment]
According to the discussion [1], we would like to accept English documents on Wiki. 
You can find some discussion about the MPP query engine on the mailing list.

[1] https://lists.apache.org/thread/5h813dlhct0c3ly2gx9h3yr8c4rfprz4

**Reply by hoho1221** [reply]
agree that, but removing Chinese docs on Wiki before English docs are completed maybe is not a good idea

**Reply by jixuan1989** [reply]
yes.. I  admit the solution is not very good. The community also has the voice.. 
  However, we tried to encourage contributors to supply English while still accepting the Chinese documents last year. The result is... few contributors had a change... 
  The same thing occurs on Apache Skywalking, which finally does not accept Chinese documents before the English documents are done. 
  IMO, this is just a temporary phase.  Finally we will leave the decision to contributors.

**Reply by hoho1221** [reply]
Hi, this doc is very much needed, reading codes without design docs makes me so confused, can you please make this doc open for us, thanks.
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=199531628

**Reply by jixuan1989** [reply]
there is no private documents on Confluence.

**Reply by hoho1221** [reply]
oh... maybe it was deleted, do we have another doc related to the design of MPP module?
thx.
